common/src/main/java/org/wso2/carbon/testgrid/common/constants/TestGridConstants.java

```diff
@@ -13,6 +13,5 @@
     public static final String TEST_TYPE_TESTNG = "TESTNG";
 
     public static final String JMTER_SUFFIX = ".jmx";
-
-
+    public static final String TESTNG_SUFFIX = ".jar";
```

Comment:
Is this the suffix of testng jar? If not the constant name is incorrect imo.

Reply:
Yes it is.